src/components/AppDirector/index.js

```diff
@@ -97,7 +97,7 @@ export default class AppDirector extends PureComponent {
                 style={{ width: '100%' }}
               >
                 {members.map((d) => (
-                  <Option key={d.user_name}>{d.user_name}</Option>
+                  <Option key={d.nick_name}>{d.user_name}</Option>
```

Comment:
nick_name should be displayed. The code is the opposite

Reply:
The back end asks for nick_name

Reply:
Have to repair